deckhouse-controller/crds/doc-ru-module-release.yaml

```
@@ -20,6 +20,8 @@ spec:
                   description: Время, до которого отложено обновление.
                 requirements:
                   description: Структура содержащая требования, необходимые для применения релиза.
+                changelog:
+                  description: Изменения модуля в данном релизе.
```

```suggestion
                  description: Список изменений модуля в данном релизе.
```
@libmonsoon-dev Это список будет список на основе PR, относящихся к модулю, да?

---

Да, все верно